book/chapters/motivation.ipynb

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
move geoweaver to another section
-geoweaver is  not the motivation to building swe worflow
-example or more info to support `slowing down progress in SWE research`
-example or more info to support the word `urgent` in second paragraph